nist-interface/src/components/mapboxViewer/config.js

```diff
@@ -0,0 +1,29 @@
+
+export const BASEMAP_STYLES = [
+    {
+      id: 'satellite',
+      label: 'Satellite',
+      mapboxId: 'cldu1cb8f00ds01p6gi583w1m',
+      thumbnailUrl: `https://api.mapbox.com/styles/v1/covid-nasa/cldu1cb8f00ds01p6gi583w1m/static/-9.14,38.7,10.5,0/480x320?access_token=${process.env.MAPBOX_TOKEN}`
+    },
+    {
+      id: 'dark',
+      label: 'Default dark',
+      mapboxId: 'cldu14gii006801mgq3dn1jpd',
+      thumbnailUrl: `https://api.mapbox.com/styles/v1/mapbox/dark-v10/static/-9.14,38.7,10.5,0/480x320?access_token=${process.env.MAPBOX_TOKEN}`
+    },
+    {
+      id: 'light',
+      label: 'Default light',
+      mapboxId: 'cldu0tceb000701qnrl7p9woh',
+      thumbnailUrl: `https://api.mapbox.com/styles/v1/mapbox/light-v10/static/-9.14,38.7,10.5,0/480x320?access_token=${process.env.MAPBOX_TOKEN}`
+    },
+    {
+      id: 'topo',
```

Comment:
aren't these default configurations in mapbox? do we need to create our own?

Reply:
Except for the `covid-nasa` thumbnailUrl others are default. It was placed there as a reference. Will remove it of redundancy.